sample/src/androidTest/java/com/stepstone/stepper/sample/SetButtonColorProgrammaticallyActivityTest.java

```diff
@@ -0,0 +1,88 @@
+package com.stepstone.stepper.sample;
+
+import android.support.test.filters.LargeTest;
+
+import com.stepstone.stepper.internal.widget.StepTabStateMatcher;
+import com.stepstone.stepper.sample.test.action.SpoonScreenshotAction;
+
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.doubleClick;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static com.stepstone.stepper.sample.test.matcher.CommonMatchers.checkCompleteButtonShown;
+import static com.stepstone.stepper.sample.test.matcher.CommonMatchers.checkCurrentStepIs;
+import static com.stepstone.stepper.sample.test.matcher.CommonMatchers.checkTabState;
+import static com.stepstone.stepper.test.StepperNavigationActions.clickComplete;
+import static com.stepstone.stepper.test.StepperNavigationActions.clickNext;
+import static org.hamcrest.Matchers.allOf;
+
+/**
+ * Performs tests on a stepper on which we change the button colors in code.
+ *
+ * @author Piotr Zawadzki
+ */
+@LargeTest
+public class SetButtonColorProgrammaticallyActivityTest extends AbstractActivityTest<SetButtonColorProgrammaticallyActivity> {
```

Comment:
What do you think about actually testing tinting with custom Matcher in Espesso tests?

Reply:
Good idea, I'll see what can be done :)

Reply:
Done :)